packages/core/src/crypto/jose/jwk/ecCompression.ts

```diff
@@ -11,9 +11,10 @@ const curveToPointLength = {
   'P-256': 64,
```

Comment:
I think same here maybe. Would be good to use JwaCurve values here maybe?

Reply:
Added JwaCurve values